templates/base.html

```diff
@@ -34,7 +35,19 @@
 
             <!-- Footer -->
             <div class="footer">
-                <p><a href="{% url 'privacy' %}">Privacy</a></p>
+            <ul class="footer-items">
+                <li><a href="{% url 'privacy' %}">Privacy</a></li>
+                <li><a href="{% url 'terms' %}">Terms of Use</a></li>
+                <li><a href="{% url 'trust' %}">Trust & Safety</a></li>
```

Comment:
The ampersand is highlighted here because `&` on its own is an invalid HTML character. What you mean here is the HTML code `&amp;`:
```html
<li><a href="{% url 'trust' %}">Trust &amp; Safety</a></li>
```